[2025-09-02T17:51:46.255+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-02T17:51:46.258+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_pipeline_dag.wrangle_flight_data scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T17:51:46.260+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_pipeline_dag.wrangle_flight_data scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T17:51:46.260+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-09-02T17:51:46.262+0530] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): wrangle_flight_data> on 2025-09-01 00:00:00+00:00
[2025-09-02T17:51:46.267+0530] {logging_mixin.py:190} WARNING - /Users/parthmac/Desktop/Projects/Flight Data Engineering/flight-env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=33255) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-02T17:51:46.267+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'flight_pipeline_dag', 'wrangle_flight_data', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/flight_etl_dag.py', '--cfg-path', '/var/folders/kk/d6w7_wpx76q43_lgsgr3g6x40000gn/T/tmpq1ac4gn_']
[2025-09-02T17:51:46.268+0530] {standard_task_runner.py:72} INFO - Started process 33258 to run task
[2025-09-02T17:51:46.268+0530] {standard_task_runner.py:105} INFO - Job 8: Subtask wrangle_flight_data
[2025-09-02T17:51:46.280+0530] {task_command.py:467} INFO - Running <TaskInstance: flight_pipeline_dag.wrangle_flight_data scheduled__2025-09-01T00:00:00+00:00 [running]> on host parths-macbook-air.local
[2025-09-02T17:51:46.295+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='noemail@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='flight_pipeline_dag' AIRFLOW_CTX_TASK_ID='wrangle_flight_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-09-02T17:51:46.295+0530] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-09-02T17:51:46.295+0530] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-09-02T17:51:46.295+0530] {logging_mixin.py:190} INFO - Current task name:wrangle_flight_data state:running start_date:2025-09-02 12:21:46.258743+00:00
[2025-09-02T17:51:46.295+0530] {logging_mixin.py:190} INFO - Dag name:flight_pipeline_dag and current dag run status:running
[2025-09-02T17:51:46.295+0530] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-02T17:51:46.296+0530] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/kk/d6w7_wpx76q43_lgsgr3g6x40000gn/T
[2025-09-02T17:51:46.296+0530] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "python '/Users/parthmac/Desktop/Projects/Flight Data Engineering/extraction and wrangling/data_wrangling.py'"]
[2025-09-02T17:51:46.299+0530] {subprocess.py:86} INFO - Output:
[2025-09-02T17:51:46.396+0530] {subprocess.py:93} INFO - An error occurred: [Errno 2] No such file or directory: 'data/raw data/flights_raw_data.json'
[2025-09-02T17:51:46.408+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-02T17:51:46.412+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-02T17:51:46.413+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flight_pipeline_dag, task_id=wrangle_flight_data, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20250902T122146, end_date=20250902T122146
[2025-09-02T17:51:46.415+0530] {logging_mixin.py:190} INFO - Task instance in success state
[2025-09-02T17:51:46.415+0530] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-09-02T17:51:46.415+0530] {logging_mixin.py:190} INFO - Dag name:flight_pipeline_dag queued_at:2025-09-02 12:21:38.268997+00:00
[2025-09-02T17:51:46.415+0530] {logging_mixin.py:190} INFO - Task hostname:parths-macbook-air.local operator:BashOperator
[2025-09-02T17:51:46.446+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-02T17:51:46.452+0530] {taskinstance.py:3900} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-09-02T17:51:46.453+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
